packages/ember-runtime/tests/mixins/enumerable_test.js

```diff
@@ -108,6 +108,20 @@ QUnit.test('should apply Ember.Array to return value of uniq', function() {
   equal(EmberArray.detect(y), true, 'should have mixin applied');
 });
 
+QUnit.test('uniqBy', function() {
```

Comment:
These two tests should be wrapped in an `if (isEnabled('ember-runtime-computed-uniq-by')) { }`